src/main/java/com/sfuentes/Zoo.java

```diff
@@ -0,0 +1,27 @@
+package com.sfuentes;
+
+public class Zoo {
+
+  public String calculatePercentage(int[] agesAnimals) {
+    double countBetweenZeroAndOne = 0;
+    double countLessThanThree = 0;
+    double countGreaterThanOrEqualToThree = 0;
+
+    for (int ageAnimal : agesAnimals) {
+      if (ageAnimal <= 1) {
+        countBetweenZeroAndOne++;
+      } else if (ageAnimal < 3) {
+        countLessThanThree++;
+      } else {
+        countGreaterThanOrEqualToThree++;
+      }
+    }
+
+    int percentageBetweenZeroAndOne = (int) Math.round((countBetweenZeroAndOne / agesAnimals.length) * 100);
+    int percentageLessThanThree = (int) Math.round((countLessThanThree / agesAnimals.length) * 100);
+    int percentageGreaterThanOrEqualToThree = (int) Math.round((countGreaterThanOrEqualToThree / agesAnimals.length) * 100);
+
+    return "1.Category 0-1: " + percentageBetweenZeroAndOne + "%, 2.Category 1-3: " + percentageLessThanThree + "%, 3.Category 3 or more: " + percentageGreaterThanOrEqualToThree + "%";
```

Comment:
Instead of returning a string you could return a Map like a HashMap<String, Integer> for the key(String) would be the category name and the value(Integer) would be the percentage.
if you haven't work with hashmaps watch this video https://www.youtube.com/watch?v=-JOSjIan2g0
and if you got stuck you can ask me anytime.
And please note that if you change this you will need to change the way you test this method

Reply:
Fixed.